client/start.go

```diff
@@ -8,12 +8,8 @@ import (
 )
 
 func (cli *HyperClient) HyperCmdStart(args ...string) error {
-	var opts struct {
-		Cpu int `short:"c" long:"cpu" default:"1" value-name:"1" description:"CPU number for the VM"`
-		Mem int `short:"m" long:"memory" default:"128" value-name:"128" description:"Memory size (MB) for the VM"`
-	}
-	var parser = gflag.NewParser(&opts, gflag.Default)
-	parser.Usage = "start [-c 1 -m 128]| POD_ID \n\nLaunch a 'pending' pod"
+	var parser = gflag.NewParser(nil, gflag.Default)
+	parser.Usage = "start POD_ID [CONTAINER_ID]\n\nLaunch a created pod or container"
```

Comment:
podId is not required when starting a created container, add a new `-c` option for containers?